src/Resources/skeleton/doctrine/Entity.tpl.php

```diff
@@ -9,29 +9,39 @@
 <?php if ($broadcast): ?>use Symfony\UX\Turbo\Attribute\Broadcast;
 <?php endif ?>
 
+<?php if(!$use_attributes || !$doctrine_use_attributes): ?>
```

Comment:
Small nitpick (other occurrences should be fixed as well)
```suggestion
<?php if (!$use_attributes || !$doctrine_use_attributes): ?>
```